api/api/src/main/scala/uk/ac/wellcome/platform/api/elasticsearch/ElasticsearchQuery.scala

```diff
@@ -156,6 +172,20 @@ final case class IdQuery(q: String) extends ElasticsearchPartialQuery {
     )
 }
 
+final case class ImageIdQuery(q: String) extends ElasticsearchPartialQuery {
+  lazy val idFields = Seq(
+    "id.canonicalId.text",
+    "id.sourceIdentifier.value.text",
+    "parentWork.text"
+  )
```

Comment:
Having this defined as `lazy` is a little confusing , as it is currently a just constant that will be included as some bytes in the jar

Reply:
Yeah, I did it to keep consistency with the other classes but I think I'll remove it from all of them 👍

Reply:
No idea why I'd done this...